src/app/upgrade.cpp

```diff
@@ -229,12 +232,23 @@ namespace
 
 bool upgrade(const bool /*ask*/)
 {
-    exportWebUIHttpsFiles();
-    upgradeTorrentContentLayout();
-    upgradeListenPortSettings();
-    upgradeSchedulerDaysSettings();
-    upgradeDNSServiceSettings();
-    upgradeTrayIconStyleSettings();
+    CachedSettingValue<int> version {"MigrationVersion", 0};
```

Comment:
Shouldn't we mark such service records, e.g. `@MigrationVersion`?

Reply:
I'm not familiar with `@` prefix, what is it for? Is it commonly used elsewhere?

Reply:
>I'm not familiar with @ prefix, what is it for?

It's just example. It can be any "sign" to differentiate such an internal/service record when editing manually, e.g.: `__MigrationVersion`, `$MigrationVersion$` etc.

Reply:
Then instead of adding a prefix, it may be better to place it under another section, for example: `Internal/MigrationVersion`. Currently (by default) it is under `General/MigrationVersion`.

Reply:
> Then instead of adding a prefix, it may be better to place it under another section, for example: `Internal/MigrationVersion`.

Ok. Or `Meta/MigrationVersion`

Reply:
>Or Meta/MigrationVersion

Done.